docs/topics/gradle/gradle-backend-tutorial.md

```diff
@@ -0,0 +1,107 @@
+      JDK home directory.
+    * If you don't have the necessary JDK on your computer, select **Download JDK**.
+
+7. Enable the **Add sample code** option to create a file with a sample `"Hello World!"` application.
+8. Click **Create**.
+
+## Understand the build script
+
+Open the `build.gradle.kts` file: it is the Gradle Kotlin build script, which contains a list of the dependencies required for the application.
+
+Here is the full script with the explanation of all parts and dependencies:
+
+```kotlin
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile // For `KotlinCompile` task below
+
+plugins {
+    kotlin("jvm") version "%kotlinVersion%" // The version of Kotlin to use
+    application // The application plugin to add support for building a CLI application in Java
+}
+
+group = "org.example" // Usually a company name, for example, `org.jetbrains`
+version = "1.0-SNAPSHOT" // A version that will be assigned to the built artifact
+
+repositories { // A source of dependencies
+    mavenCentral() // Often, it's `mavenCentral`. Also, it can be a private repository where your company stores artifacts
+}
+
+dependencies { // All the libraries you want to use
+   // You can copy dependencies' names after you find them at the [Maven Central Repository](https://search.maven.org/)
+   testImplementation(kotlin("test")) // The Kotlin test library
+}
+
+tasks.test {
+    useJUnitPlatform() // [JUnitPlatform for tests](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html#useJUnitPlatform)
+}
+
+tasks.withType<KotlinCompile> { // Settings for `KotlinCompile` tasks
+   // Kotlin compiler options
+    kotlinOptions.jvmTarget = "1.8" // This option specifies the target version of the generated JVM bytecode
+}
+
+application {
+    mainClass.set("MainKt") // Defining the main class for the application
+}
+```
+
+As you can see, there are a few Kotlin-related artifacts added to the Gradle build file:
+
+1. In the `plugins` block, there is the `kotlin("jvm")` artifact – the plugin defines the version of Kotlin to be used in the project.
+
+2. In the `dependencies` section, there is `testImplementation(kotlin("test"))`. 
+   Learn more about [setting dependencies on test libraries](gradle-configure.md#set-dependencies-on-test-libraries).
+
+3. After the dependencies section, you can see the `KotlinComiple` task configuration block.
+   This is where you can add extra arguments to the compiler to enable or disable various language features.
+
+## Run the application
+
+Open the `Main.kt` file in `src/main/kotlin`.  
+The `src` directory contains Kotlin source files and resources. The `Main.kt` file contains sample code that will print
+`Hello World!`.
+
+![Main.kt with main fun](jvm-main-kt-initial-gradle.png){width=700}
+
+The easiest way to run the application is to click the green **Run** icon in the gutter and select **Run 'MainKt'**.
```

Comment:
As I understood, this UI is called gutter and we use this word in other parts of our documentation.

Reply:
This may be something for a bigger discussion elsewhere then. "Gutter" makes me think of a drainpipe. I've never heard/seen it used as a name for a UI element before.

Reply:
You can get familiar with this here https://www.jetbrains.com/help/idea/settings-gutter-icons.html

Reply:
This is good to know! I'll set this to resolved as no change is needed.